### ISSUE TITLE
Cannot configure path to external code editor for C# files

### ISSUE THREAD

**Issue by Magdiel012** [godotengine/godot#66904]
### Godot version

v4.0.beta1.mono.official [4ba934bf3]

### System information

Windows 10

### Issue description

Currently it seems Godot expects the path to the selected external code editor in `mono/editor/external_editor` to have been configured by an installer automatically, which is not the case when running the editor portably.

### Steps to reproduce

Attempt to configure an external code editor in `Mono > Editor` that hasn't been installed to the system, but is instead running portably (I tested with VSCode, but presumably this applies to other editors that don't require installation as well).

**Comment by MagdielM** [comment]
> I don't use a portable editor, I use VSCode and have it installed. However, I disabled the External Editor option under `Mono > Editor`. I made the settings under `Text Editor > External` and it works for me. Did you also set the settings under `Text Editor > External`?

This would work, but I'd personally like to be able to edit GDScript from the Godot editor and use an external editor for C# code.

**Comment by Calinou** [comment]
> that hasn't been installed to the system, but is instead running portably

This is expected. The system isn't aware of where you've placed the portable VS Code installation, so you **must** specify the path to the executable.

**Comment by Magdiel012** [comment]
> > that hasn't been installed to the system, but is instead running portably
> 
> This is expected. Portable installations are by design not registered on the system in any way. This means the system isn't aware of where you've placed the portable VS Code installation, so you **must** specify the path to the executable.

Indeed, the issue being that there is no option to do this for the Mono-specific external editor.
![image](https://user-images.githubusercontent.com/114442112/194624683-bc5372dd-45a9-45c3-a7d0-3d9fcde3deef.png)
![image](https://user-images.githubusercontent.com/114442112/194624732-60445216-a880-4d77-8e33-306a51b47d39.png)
